app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java

```diff
@@ -134,16 +163,16 @@ private UploadItem getUploadItem(UploadableFile uploadableFile,
             createdTimestampSource = dateTimeWithSource.getSource();
         }
         Timber.d("File created date is %d", fileCreatedDate);
-        GPSExtractor gpsExtractor = fileProcessor
-                .processFileCoordinates(similarImageInterface, context);
-        UploadItem uploadItem = new UploadItem(uploadableFile.getContentUri(),
-                Uri.parse(uploadableFile.getFilePath()),
+        GPSExtractor gpsExtractor = fileProcessor.processFileCoordinates(similarImageInterface, context);
```

Comment:
Un related

Reply:
GPSExtractor was not added by me It might be a result due to some other merges to this PR.
Should I remove this?

Reply:
You don't have to remove it, just revert the formatting changes in these two lines

Reply:
Ok Sorry, I misunderstood. I'll make the necessary changes